### PR TITLE
Add LC709203F, unitless percent type

### PR DESCRIPTION
**Pull request by brentru**
Adds sensor [LC709203F](https://www.adafruit.com/product/4712)
Adds new sensor type, unitless-percent

### CODE REVIEW

File: components/i2c/lc709203f/definition.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "subcomponents": [
+    {
+      "displayName": "Battery Cell Voltage",
+      "sensorType": "voltage",
+      "defaultPeriod": 900
+    },
+    {
+        "displayName": "Battery Cell Percent",
+        "sensorType": "unitless-percent",
+        "defaultPeriod": 900
```
[the remark this answers is not in the document]
Comment (lorennorman) [reply components/i2c/lc709203f/definition.json:13]: and lines 11-13 need to be outdented 2 spaces.

File: components/sensors.json [hunk 2]
```diff
@@ -110,5 +110,9 @@
   "eco2": {
     "displayName": "Estimated C02",
     "defaultPeriod": 900
+  },
+  "unitless-percent": {
+    "displayName": "Unitless Percentage",
```
Comment (lorennorman) [inline components/sensors.json:115]: Maybe should just be "Percentage"? My thinking is that "Unitless" shouldn't ever be seen by end users, but plain old "Percentage" may actually be good enough for some uses in the future. Up to you!

**Review by lorennorman — CHANGES_REQUESTED** [review]
great work! couple easy changes

File: components/sensors.json (hunk 2 above)
Comment (brentru) [reply components/sensors.json:115]: Let's keep this, humidiy can also be expressed as a descriptive percentage .

File: components/i2c/lc709203f/definition.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (brentru) [reply components/i2c/lc709203f/definition.json]: Resolved in https://github.com/adafruit/Wippersnapper_Components/pull/66/commits/09bfdb2210e17152911298bea72d2a0ffe6e0976

**Review by lorennorman — APPROVED** [review]
Cool this is ready after the other PRs go live.

**Pull request merged**